sjb/syntax.md

```diff
@@ -0,0 +1,211 @@
+# `sjb` YAML Syntax Reference
```

Comment:
Steve's Job Builder but like the gasoline company BP (British Petroleum) I'm not super happy with that acronym and  don't really want to shove my name into it